ffi/Gosu_Font.cpp

```diff
@@ -38,6 +38,13 @@ GOSU_FFI_API unsigned Gosu_Font_flags(Gosu_Font* font)
     });
 }
 
+GOSU_FFI_API unsigned Gosu_Font_image_flags(Gosu_Font* font)
+{
+    return Gosu_translate_exceptions([=] {
+        return font->font.image_flags();
+    });
+}
```

Comment:
I'm removing this because MRI Ruby/Gosu doesn't have it either, even if it is somewhat inconsistent with C++ Gosu.